composer/helpers/build

```diff
@@ -15,3 +15,5 @@ composer validate --no-check-publish
 composer install
 composer run lint -- --dry-run
 composer run stan
+
```

Comment:
Worth adding a comment explaining why its deleted here and ref updater?

Reply:
```suggestion

# Composer caches source zips and repo metadata, none of which is useful. Save space in this layer
```